README.md

```diff
@@ -63,8 +63,11 @@ If you have a solid go and node developer setup, you may skip this,
 but good to go through it to be sure.
 
 Once you are set up, you should be able to run something
-like the following to compile both `mycoind` (sample app)
-and `tendermint` (a [BFT consensus engine](https://tendermint.com)):
+like the following to compile both `bnsd` (iov blockchain application)
```

Comment:
should `iov` be `IOV`?

Reply:
Yes, but not if that makes me wait another hour in the CI queue....